lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -251,4 +251,51 @@ describe('ReportRenderer', () => {
       .querySelectorAll('.lh-audit--notapplicable').length;
     assert.strictEqual(notApplicableCount, notApplicableElementCount);
   });
+
+  describe('axe-core', () => {
+    let axe;
+
+    beforeAll(() =>{
+      // Needed by axe-core
+      // https://github.com/dequelabs/axe-core/blob/581c441c/doc/examples/jsdom/test/a11y.js#L24
+      global.window = global.self;
+      global.Node = global.self.Node;
+      global.Element = global.self.Element;
+
+      // axe-core must be required after the global polyfills
+      axe = require('axe-core');
+    });
+
+    afterAll(() => {
+      global.window = undefined;
+      global.Node = undefined;
+      global.Element = undefined;
+    });
+
+    it('renders without axe violations', (done) => {
+      const container = renderer._dom._document.body;
+      const output = renderer.renderReport(sampleResults, container);
+
+      const config = {
+        rules: {
+          // Reports may have duplicate ids
+          // https://github.com/GoogleChrome/lighthouse/pull/5058#discussion_r184791923
```

Comment:
Now we just have to remember to remove this when that issue is closed :smile: 
```suggestion
          // https://github.com/GoogleChrome/lighthouse/issues/9432
```

Reply:
Done.